src/styles/components/_product-content.scss

```diff
@@ -7,14 +7,31 @@
     'detail'
     'information';
 
-  @include respond-above('lg') {
+  @include respond-above('md') {
     grid-template-columns: repeat(2, 1fr);
     grid-template-rows: repeat(2, 1fr);
     grid-template-areas:
       'media detail'
       'information information';
   }
 
+  & > div {
```

Comment:
Since below this class we specify the selected class explicitly (ex: `& > .product-...`), this should also be specify explicitly too. For readability and also the block we're selecting is a sub-main (next level from `main`) block which is not a small element like simple `div` with one level `span` (`div > span`).

Reply:
You mean to change it to this below. 

<img width="407" alt="Screen Shot 2564-03-28 at 00 11 14" src="https://user-images.githubusercontent.com/32285706/112728488-1e1d5700-8f5a-11eb-8fd4-9b246d75866b.png">

Or this
<img width="348" alt="Screen Shot 2564-03-28 at 00 13 13" src="https://user-images.githubusercontent.com/32285706/112728543-6a689700-8f5a-11eb-841c-3fc5cd26f519.png">

If there are better ways of writing this please suggest one.

Reply:
The first one is the cleanest and readable for now. 😎